changes/624.feature.md

```diff
@@ -0,0 +1 @@
+Allow reading compute plugin blocklists from toml file.
```

Comment:
How about specifying "which" configuration to edit, instead of just toml?
```suggestion
Allow reading compute plugin blocklists from agent local configuration.
```